src/components/grid/GridComponent.vue

```diff
@@ -383,4 +391,12 @@ export default Vue.extend({
     }
   }
 
+  .no-results {
+    color: $secondary;
+    font-size: 2rem;
+    font-weight: 600;
+    margin-top: 150px;
```

Comment:
suggest to use rems , do we need a specific style for this, could we not use a style from the theme and use bootstrap helper classes for placement

Reply:
I checked, but there are no out-of-the-box solutions for this. Added custom styling.